central/reports/scheduler/schedule.go

```diff
@@ -72,12 +73,16 @@ var (
     }`
 
 	vulnReportEmailTemplate = `
-	Red Hat Advanced Cluster Security for Kubernetes has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report {{.WhichVulns}} for {{.DateStr}}.
+	%s has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report {{.WhichVulns}} for {{.DateStr}}.
 
 	To address these findings, please review the impacted software packages in the container images running within deployments you are responsible for and update them to a version containing the fix, if one is available.`
 
 	noVulnsFoundEmailTemplate = `
-	Red Hat Advanced Cluster Security for Kubernetes has found zero vulnerabilities associated with the running container images owned by your organization.`
+	%s has found zero vulnerabilities associated with the running container images owned by your organization.`
+
+	rhacsBranding = `Red Hat Advanced Cluster Security for Kubernetes`
+
+	stackroxBranding = `StackRox`
```

Comment:
These things may be more appropriate closer to the definition of the environment variable. Something like (pseudocode):
```
func getBrandedProductName() string {
   if (doesEnvVarSetRHACSBranding?()) {
    return "Red Hat ..."
  } else {
    return "StackRox"
  }
}
```

Reply:
Okay, I will create env variables for these.